new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -119,24 +119,24 @@
 
   /**
    * Install Operator.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
```

Comment:
i would be find to say `@param namespaces injected by JUnit extension`

Reply:
Modified it.